java-client/src/main/java/org/scassandra/http/client/ActivityClient.java

```diff
@@ -51,7 +51,7 @@
         private String host = "localhost";
 
         private int port = 8043;
-        private int socketTimeout = 1000;
+        private int socketTimeout = 5000;
```

Comment:
Bumping of timeouts seems a bit ominous, but the main reason I did this was the window was pretty tight previously and tests would occasionally fail because of socket read timeouts.  Could never reproduce the timeout issue locally, but happened from time to time in travis.